lib/replicate.js

```diff
@@ -682,9 +683,8 @@ Peer.prototype._request = function (index, bytes, hash) {
     nodes: this.feed.digest(index)
   }
 
-  if (this._requestTimeout === null) {
-    const ms = this.stream.stream.timeout ? this.stream.stream.timeout.ms : 20000
-    this._requestTimeout = timeout(ms, this._onrequesttimeout, this)
+  if (this.stream.stream.timeout) {
```

Comment:
this still needs the null check unless i'm missing something?

Reply:
oops, good catch!